packages/backend-core/src/Base.ts

```diff
@@ -88,9 +94,10 @@ export class Base {
    * The public key will be supplied in the form of CryptoKey or will be loaded from the CLERK_JWT_KEY environment variable.
    *
    * @param {string} token
+   * @param {string[]} authorizedParties
```

Comment:
🙃 
I think this should actually be {VerifySessionTokenOptions} verifySessionTokenOptions

Reply:
Fixed in [02cd6d8](https://github.com/clerkinc/javascript/pull/15/commits/02cd6d819029c3c66f8e308028e81c5731841cd8)